README.md

```diff
@@ -2294,7 +2294,7 @@ Pass `-help` to VictoriaMetrics in order to see the list of supported command-li
   -envflag.prefix string
      Prefix for environment variables if -envflag.enable is set
   -eula
-     By specifying this flag, you confirm that you have an enterprise license and accept the EULA https://victoriametrics.com/assets/VM_EULA.pdf . This flag is available only in VictoriaMetrics enterprise. See https://docs.victoriametrics.com/enterprise.html
+     By specifying this flag, you confirm that you have an enterprise license and accept the ELA https://victoriametrics.com/legal/esa/ . This flag is available only in VictoriaMetrics enterprise. See https://docs.victoriametrics.com/enterprise.html
```

Comment:
Should it be ESA instead of ELA?

Reply:
yeap, thank you

Reply:
fixed in https://github.com/VictoriaMetrics/VictoriaMetrics/commit/7be3848ee6f06cc90e17331f300b7135c64fb376